lib/jekyll/jekyll-sitemap.rb

```diff
@@ -8,24 +8,25 @@ class JekyllSitemap < Jekyll::Generator
     # Main plugin action, called by Jekyll-core
     def generate(site)
       @site = site
-      @site.config["time"]         = Time.new
-      @site.config["html_files"]   = html_files.map(&:to_liquid)
+      @site.config["time"] = Time.new
+      @site.config["sitemap_files"] = sitemap_files.map(&:to_liquid)
```

Comment:
`sitemap_files` (or `sitemap.files`) isn't sitting right with me. I don't know that we're really describing what we've got there. Static files? Additional files? External files? moar files?